en/companion_computer/README.md

```diff
@@ -0,0 +1,61 @@
+# Companion Computers
+
+Companion computers ("mission computers"), are separate on-vehicle computers that enable computationally expensive features like [object avoidance](../computer_vision/obstacle_avoidance.md) and [collision prevention](../computer_vision/collision_prevention.md).
+
+The diagram below shows a possible architecture for an unmanned vehicle architecture that includes a flight controller and companion computer.
+
+![PX4 architecture - FC + Companion Computer](../../assets/diagrams/px4_companion_computer_simple.svg)
+
+<!-- source for drawing: https://docs.google.com/drawings/d/1ZDSyj5djKCEbabgx8K4ESdTeEUizgEt8spUWrMGbHUE/edit?usp=sharing -->
+
+The flight controller runs PX4 on NuttX, and provides core flight and safety code.
+The companion computer usually runs Linux, as this is a much better platform for "general" software development.
+They are connected using a fast serial or Ethernet link, and typically communicate using the [MAVLink protocol](https://mavlink.io/en/) or microROS/microDDS.
```

Comment:
I probably wouldn't add microROS to avoid confusion

Reply:
Thanks. Fixed.